### PR TITLE
Adding trigger_all_talos_jobs ability in pulse_actions.

### PR DESCRIPTION
**Pull request by vaibhavmagarwal**
This will depend on https://github.com/armenzg/mozilla_ci_tools/pull/304
Will need to release new mozci version before this gets merged in.

### CODE REVIEW

File: pulse_actions/handlers/treeherder_resultset.py [hunk 1]
```diff
@@ -18,7 +19,12 @@ def on_resultset_action_event(data, message, dry_run):
             status = 'trigger_missing_jobs request sent'
         else:
             status = 'Dry-mode, no request sent'
-
+    elif action == "trigger_all_talos_jobs":
+        trigger_all_talos_jobs(repo_name, revision, times, dry_run=dry_run)
+        if not dry_run:
+            status = 'trigger_all_talos_jobs request sent'
```
Comment (adusca) [inline pulse_actions/handlers/treeherder_resultset.py]: Could you add 'times' to the status message?

**Comment by adusca** [comment]
r+ after the small times thing. But I would like to wait until automatic backfilling gets merged into master to merge and deploy this, because working from one branch and deploying from another adds some overhead.

**Comment by vaibhavmagarwal** [comment]
I have changed the version and updated the nit. This patch is working fine locally.
@adusca merge this whenever you find it right (after the automatic backfilling).

**Comment by adusca** [comment]
It looks like there was a conflict when I merged the automatic backfilling PR. Can you fix it? Feel free to merge after solving the merge conflict.

**Comment by vaibhavmagarwal** [comment]
Rebased on master branch.

**Pull request merged**